doc/source/docs/user_guide/projections.rst

```diff
@@ -465,11 +465,11 @@ Or to check if a CRS was a certain UTM zone:
 
    '+proj=utm ' in gdf.crs
 
-could be replaced with the longer but more robust check:
+could be replaced with the more robust check (requires pyproj 2.6+):
 
 .. code-block:: python
 
-   gdf.crs.is_projected and gdf.crs.coordinate_operation.name.upper().startswith('UTM')
+   gdf.crs.utm_zone
```

Comment:
The other check (and previous version of this) returned a bool. Maybe make this `gdf.crs.utm_zone is not None`?

Reply:
Sounds reasonable.